Assets/Scripts/ParentSlideController.cs

```diff
@@ -17,22 +21,24 @@ public class ParentSlideController : MonoBehaviour
     // Broadcast to all children to become visible
     private void Start()
     {
-        if(!isDisappearing)
+        transform.BroadcastMessage("SetMaterial", new Material[2] {transparentMaterialOverwrite, opaqueMaterialOverwrite});
+
+        if(isDisappearing)

```

Comment:
Should it be isVisible? Or is it starting to disappear? Not an issue really

Reply:
Since the boolean isDisappearing indicates if a specific slide instance is meant to cycle between visible and invisible states isVisible is not really fitting either. New name could be good, not sure what.